mmv1/third_party/terraform/services/networksecurity/resource_network_security_firewall_endpoint_association_test.go.erb

```diff
@@ -50,66 +111,71 @@ func TestAccNetworkSecurityFirewallEndpointAssociations_basic(t *testing.T) {
 	})
 }
 
-func testAccNetworkSecurityFirewallEndpointAssociation_basic(randomSuffix string, orgId string) string {
+func testAccNetworkSecurityFirewallEndpointAssociation_basic(randomSuffix string, orgId string, billingProjectId string, projectNumberId string) string {
 	return fmt.Sprintf(`
 resource "google_compute_network" "foobar" {
-	provider                = google-beta
-    name                    = "tf-test-my-vpc%s"
+	provider                  = google-beta
+    name                    = "tf-test-my-vpc%[2]s"
```

Comment:
Can you switch to `context` / `Nprintf`instead of Sprintf? This will be much easier to read using that formatting instead.